src/reducers/auth-reducer.js

```diff
@@ -0,0 +1,84 @@
+/**
+ *
+ * @param {Object} loginState Defines the current state of login
+ * @param {*} param1 type: action to perform, payload: Defines the state to be updated
+ * @returns {Object} Updated loginState
+ */
```

Comment:
nice way to add the info here also add the definition of function before param(just a suggestion)